charts/workflows/templates/workflow-label-clusterpolicy.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: {{ .Release.Name }}-posix-uid-label
+spec:
+  background: false
+  validationFailureAction: Enforce
+  rules:
+    - name: apply-posix-uid-label
+      match:
+        resources:
+          kinds:
+            - argoproj.io/*/Workflow
+      mutate:
+        patchStrategicMerge:
+          metadata:
+            labels:
+              workflows.argoproj.io/creator-posix-uid: '{{ `{{ request.userInfo.extra | "workflows.diamond.ac.uk/posixuid" | [0] }}` }}'
```

Comment:
We should use a `workflows.diamond.ac.uk` scoped label here, as it's nothing to do with `workflows.argoproj.io`

Reply:
Done